### DISCUSSION TITLE
Folder path in title bar?

### DISCUSSION THREAD

**Discussion by RobertCorner38** [zim-desktop-wiki/zim-desktop-wiki#1783] [Q&A] [answered]
Hello & greetings to all,
I've been using Zim for over a year now and am very happy with the software. Many thanks to Jaap (and others?) for this excellent tool. I shall definitely be contributing financially soon.

Can someone tell me please if it's possible *to show the folder path of the current notebook in the titlebar, or as a pop-up on the tray icon entry,  or elsewhere*?

Explanation : I have several notebooks, and I keep duplicates of them. They are often open at the same time. I work from several machines and am still experimenting on how to access the same notebooks from these devices (cloud, usb stick, syncthing+Markor).  This means that for the moment I do not have a stable routine, as regards which files I use (usb or hard drive) or how I open them. Once I have opened a notebook, *how do I verify at a glance where I am working* (without using the Open folder item on the Tools menu) ? The path in the title bar would be great...

I'm on MxLinux (Debian 10) and Windows 10, Zim 0.68 & 0.74.

Many thanks for your help
Robert

**Comment by jaap-karssenberg** [comment, marked as answer]
Sounds like you have the same request as in discussion #1660
